src/Giraffe/Serialization.fs

```diff
@@ -14,7 +15,7 @@ module Json =
     open Newtonsoft.Json.Serialization
     open FSharp.Control.Tasks.V2.ContextInsensitive
     open Utf8Json
-
+    let recyclableMemoryStreamManager = RecyclableMemoryStreamManager()
```

Comment:
Probably not for this PR, but should we be using this everywhere in place of `MemoryStream`?

Reply:
I'd say yes unless you know for sure what what you're putting into the Stream is going to be smaller than 85000 bytes. Since it looks like the other places this is used is in other serialization I'd say definitely can be used there.  For the tests it's not really a concern.